apps/prairielearn/src/migrations/20250206202509_variants__authn_user_id__nonnull.sql

```diff
@@ -0,0 +1,33 @@
+-- We had a few old rows in the `variants` table where `authn_user_id` was NULL.
+-- We'll copy the values over from `variants.user_id` which is always non-NULL in the rows we care about.
+UPDATE variants
+SET
+  authn_user_id = user_id
+WHERE
+  authn_user_id IS NULL;
```

Comment:
All of the rows with NULL in `authn_user_id` are old (2017) so this shouldn't block any current activity.